ViewUser.js

```diff
@@ -45,6 +45,10 @@ class ViewUser extends React.Component {
       type: 'okapi',
       path: 'users/:{id}',
       clear: false,
+      shouldRefresh: (resource, action) => {
```

Comment:
I get that this _does_ work, but I'm not entirely sure how. What does the `action` object typically look like? If we're refreshing `selUser`, why not also `loansHistory` and `permissions`, which also include `:{id}` in their paths?

Reply:
Thanks for looking into this @zburke! 

The action is being built here:

https://github.com/folio-org/stripes-connect/blob/master/epics/mutations.js#L19

and we only really care about the `action.meta` (I may change it to just pass that to `shouldRefresh` instead of a full action object).

the `meta` contains some basic info from the resource which executed the mutation (currently `path` and `name`).

The match by path currently only checks the beginning of the string with `startsWith` so the `permissions` and `loansHistory` won't match even though they include `:{id}`.